src/main/java/techreborn/blockentity/storage/item/StorageUnitBaseBlockEntity.java

```diff
@@ -377,13 +413,50 @@ public void onBreak(World world, PlayerEntity playerEntity, BlockPos blockPos, B
 		// Inventory gets dropped automatically
 	}
 
+	// The int methods are only for ContainerBuilder.sync()
```

Comment:
You should be able to sync booleans?

Reply:
Even better would be sycning the locked type so the user can see what type of item the container has been locked to.

Reply:
I have no idea how this `sync()` business works or why it's needed, I just copied it from the two other block entities that also have a lock button in their GUI (autocraft table and rolling machine). Will have to look into it.